### PR TITLE
Mods campaign by

### PR DESCRIPTION
**Pull request by BastienSaulnier**

### CODE REVIEW

**Review by LeoDiduch — DISMISSED** [review]
SnE

File: frontend/src/pages/CampaignInfos.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      </div>
+      <div className="campaignSlides"></div>
+      <div className="campaignDonation center">
+        <p>Je participe en donnant</p>
+        <ul>
+          <li>
+            <button>{campaignInfo.value1}</button>
+          </li>
+          <li>
+            <button>{campaignInfo.value3}</button>
```
Comment (LoicBrassart) [inline frontend/src/pages/CampaignInfos.jsx]: "value2", isn't it ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
There's a typo, fix it quickly and notify me on Slack

File: frontend/src/pages/CampaignInfos.jsx (hunk 1 above)
Comment (BastienSaulnier) [reply frontend/src/pages/CampaignInfos.jsx]: Yep, you are right, my bad

File: frontend/src/App.js [hunk 2]
```diff
@@ -2,15 +2,16 @@ import React from "react";
...
 import Ambassadors from "./pages/Ambassadors";
 import AmbassadorInfos from "./pages/AmbassadorInfos";
 import Causes from "./pages/Causes";
 import CauseInfos from "./pages/CauseInfos";
 import Footer from "./components/Footer";
 import "./App.scss";
-import CampaignInfos from "./pages/CampaignInfos";
 const { siteTitle } = require("./conf");
 
+
```
Comment (LoicBrassart) [inline frontend/src/App.js:14]: Why is this file touched ? 
And why is there a second empty line ? It seems to indicate that your Prettier doesn't work on `.js` files

**Review by LoicBrassart — APPROVED** [review]
SnE

**Review by LeoDiduch — APPROVED** [review]
SnE

**Pull request merged**